### PR TITLE
Location scripts as a module

### PR DESCRIPTION
**Pull request by Kert**

### CODE REVIEW

**Comment by jkoppel** [comment]
Nice work!

File: maps/MODULES/locations_1.lua [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+			GrantArmy(h, CREATURE_CYBER_KOBOLD_SPEARMAN, goblins)
+		end
+		-- someone else
+		if pikemen > 0 then
+			TakeTroop(h, CREATURE_PIKEMAN, pikemen)
+			GrantArmy(h, CREATURE_CYBER_PLASMA_LANCER, pikemen)
+		end
+		PlaySoundEffect('EXPERNCE')
+		if goblins > 0 and pikemen > 0 then
+			AdvancedMessageBox("{Hill Fort}\n\nAll of your Pikemen and Goblins have been upgraded to Cyborg Plasma Lancers and Cyborg Kobold Spearmen", 1, -1, -1, -1, -1, -1, -1, -1, 0)
```
Comment (jkoppel) [inline maps/MODULES/locations_1.lua:23]: Can you show an image of these lancers and spearmen?

https://github.com/jkoppel/project-ironfist/blob/master/maps/MODULES/AdvMessageBox.lua#L12
http://projectironfist.pbworks.com/w/page/134600181/AdvancedMessageBox%20Documentation

Similar for the other branches.

**Comment by jkoppel** [comment]
Great. Send a screenshot of the dialog box working, and then merge.

**Comment by Kert** [comment]
dialogbox

**Pull request merged**